### PR TITLE
docs: Improve markdown documentation

### PR DESCRIPTION
**Pull request by sergigraciaj**

### CODE REVIEW

File: docs/directives/markdown.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+As Margarita is built with Vue, we can can use [directives](https://vuejs.org/guide/essentials/template-syntax.html#directives),
+so to use Markdown in our components (ideally just text components as MaText, MaHeading or MaLink) we use the directive
+**v-markdown**.
+
+For more info on how to use Markdown check this [guide](https://www.markdownguide.org/).
+
+# Example of usage
+
+Code:
+`<v-text v-markdown="This is a _random_ text to show how to use the **Markdown** directive" />`
```
Comment (edufarre) [inline docs/directives/markdown.docs.mdx:17]: Shouldn't this be `ma-text`?

Comment (sergigraciaj) [reply docs/directives/markdown.docs.mdx:17]: OMG I'm very "empanado" xDD

Comment (sergigraciaj) [reply docs/directives/markdown.docs.mdx:17]: Thx!

**Review by edufarre — APPROVED** [review]

**Review by alex-castelo — APPROVED** [review]

**Pull request merged**